pkg/p2p/libp2p/internal/handshake/pb/handshake.proto

```diff
@@ -16,6 +16,7 @@ message Ack {
     BzzAddress Address = 1;
     uint64 NetworkID = 2;
     bool FullNode = 3;
+    string Transaction = 4;
```

Comment:
a tx hash is always exactly 32 bytes of data. maybe there is a better type that can be used here.

Reply:
I was looking into it but didn't find a solution.

This article says that's expected: https://www.aapelivuorinen.com/blog/2019/07/12/protobuf-arrays/